### PR TITLE
SW-376-add-config-field-cross-sell-categories

### PR DESCRIPTION
**Pull request by zaifastafa**

### CODE REVIEW

File: FinSearchUnified/Resources/config.xml [hunk 1]
```diff
@@ -27,6 +27,18 @@
             <description lang="de">Aktiviert die FINDOLOGIC Suche für Kategorieseiten.</description>
             <description lang="en">Activate the FINDOLOGIC search provider for category pages.</description>
         </element>
+        <element type="combo">
+            <name>CrossSellingCategories</name>
```
Comment (TheKeymaster) [inline FinSearchUnified/Resources/config.xml]: Please ensure that this works with all Shopware versions that we support. Shopware 5.2 should work as well.

File: FinSearchUnified/Resources/config.xml [hunk 2]
```diff
@@ -27,6 +27,18 @@
             <description lang="de">Aktiviert die FINDOLOGIC Suche für Kategorieseiten.</description>
             <description lang="en">Activate the FINDOLOGIC search provider for category pages.</description>
         </element>
+        <element type="combo">
```
Comment (TheKeymaster) [inline FinSearchUnified/Resources/config.xml]: @zaifastafa I click-tested this on my own with Shopware 5.6 and it seems to work fine. On Shopware 5.2 it also works, but I am not able to select sub-categories.

image

The moment I type anything that does not match any of those two categories (e.g. a sub category), it does not work.
Please ensure that you can also select sub-categories.

File: FinSearchUnified/Resources/config.xml [hunk 3]
```diff
@@ -27,6 +27,35 @@
...
+    proxy: {
+        type: 'ajax',
+        url: window.location.href.substr(0, window.location.href.indexOf('backend')) + 'backend/Category/getPathByQuery',
+        reader: {
+            type: 'json',
+            root: 'data',
+            totalProperty: 'total'
+        }
+    }
+    }).create();//new ]]>
```
Comment (TheKeymaster) [inline FinSearchUnified/Resources/config.xml]: I do not think that this comment `//new` is necessary.

Comment (zaifastafa) [reply FinSearchUnified/Resources/config.xml]: It is mentioned in their documentation that it is a sort of hack to load the store.

Comment (TheKeymaster) [reply FinSearchUnified/Resources/config.xml]: I see. If it works, then ok :).

**Pull request merged**